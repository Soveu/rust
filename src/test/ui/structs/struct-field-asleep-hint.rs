struct S { pre: i32 }
impl S {
    fn f(self) -> i32 {
        self.prehopjpokppppppppppppppppppp
        //~^ ERROR it seems like you've fallen asleep and hit your keyboard
    }
}

fn main() {}
