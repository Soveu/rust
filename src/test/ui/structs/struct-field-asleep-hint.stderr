error: it seems like you've fallen asleep and hit your keyboard
  --> $DIR/struct-field-asleep-hint.rs:4:17
   |
LL |         self.prehopjpokppppppppppppppppppp
   |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^ you fell asleep here
   |
   = help: try improving your sleep schedule or reducing your hours
   = note: you have been asleep for about 1.36 seconds
   = note: available fields are: `pre`

error: aborting due to previous error

